database/buildSchema/41_application_list_view.sql

```diff
@@ -46,16 +41,7 @@ CREATE OR REPLACE FUNCTION application_list (userid int DEFAULT 0)
         assigners,
         reviewers,
         reviewer_action,
-        assigner_action,
-        -- CASE WHEN is_fully_assigned_level_1 IS NULL THEN
-        --     FALSE
-        -- ELSE
-        --     is_fully_assigned_level_1
-        -- END,
-        -- assigned_questions_level_1,
-        total_questions,
-        total_assigned,
-        total_assign_locked
```

Comment:
I don't fully understand all this, but it's good to see this simplification!

Reply:
Moved to a different PR now: #879 

I'm going to make that base for this PR, but briefly explanation to you why this got removed - because it's not in use by ApplicationList (it already wasn't in use) I think this was going to be used to be checked for completeness of reviews before but stayed half-way through. The only part actually used in ApplicationList is `reviewerActions` and `assignerActions`